src/pages/Dashboard/Markets/BorrowMarket/index.tsx

```diff
@@ -77,7 +79,7 @@ export const BorrowMarketUi: React.FC<IBorrowMarketUiProps> = ({
   return (
     <div className={className} css={styles.tableContainer}>
       <Table
-        title={t('markets.borrowMarketTableTitle')}
+        title={isXlDown ? undefined : t('markets.borrowMarketTableTitle')}
```

Comment:
I'm hesitant to get in the habit of styling responsive styles in JS

Reply:
I also think it is easier to read if the ternaries return truthy values on the right and falsy values on the left

Reply:
it is made by js because we want to control this in parent scope for current case. it's not usual case for tables.

Reply:
do you mean using it like `!isXlDown ? t('markets.supplyMarketTableTitle' : undefined`

Reply:
Yeah this is an odd one because I would do like `isXLUp ? t('markets.supplyMarketTableTitle' : undefined` but we decided to have all down breakpoints.  I think the best solution here is to do it with css and not conditionally render it by subscribing to changes in the window size

Reply:
we don't want to remove titles for all tables. it's a special case, when we want to control titles in parent scope. that's why it was made here by js